tests/scripts/task_lint.sh

```diff
@@ -31,7 +31,19 @@ echo "Check file types..."
 python3 tests/lint/check_file_type.py
 
 echo "Check ASF license header..."
-java -jar /bin/apache-rat.jar -E tests/lint/rat-excludes  -d . | (grep "== File" > /tmp/$$.apache-rat.txt || true)
+
+# rat exclude patterns match specific directory entries, they cannot
+# span the / separators in paths, hence to exclude files based on
+# their position on the directory tree we use a secondary mechanism
+# that can match patterns across / seperators.
+cat > /tmp/$$.ignore-re.txt <<EOF
+== File: \./jvm/[^/]*/[^/]*/target/.*
```

Comment:
specifically, these additional ignore patterns seem complicates the checking process. It would be great to keep the original rat-exclude patterns without having to check in here

Reply:
Revised with two new patterns in rat-exclude for jvm and tutorials.